### PR TITLE
added style to buttons in example1

### PR DESCRIPTION
**Pull request by Vaibhav-28**
Added styles to increase and decrease buttons in example 1.

### CODE REVIEW

**Comment by Vaibhav-28** [comment]
I have raised a [PR](https://github.com/sonusindhu/angular-signals-examples/pull/4) for this issue

File: src/app/signal/signal-example1/signal-example1.component.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <h2>Example 1 - Count Increment/Decrement</h2>
 <p>Count is: {{ count() }}</p>
 <p>
-  <button (click)="increaseCount()">Increase</button>
-  <button (click)="decreaseCount()">Decrease</button>
-</p>
\ No newline at end of file
+  <button class="counterButton" (click)="increaseCount()">Increase</button>
```
Comment (sonusindhu) [inline src/app/signal/signal-example1/signal-example1.component.html:4]: thanks for the PR @Vaibhav-28,
could you please use the [material button](https://material.angular.io/components/button/overview) here

File: src/app/signal/signal-example1/signal-example1.component.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.counterButton {
```
Comment (sonusindhu) [inline src/app/signal/signal-example1/signal-example1.component.scss:1]: style is not required, we can use the default material button with primary color

**Review by sonusindhu — COMMENTED** [review]
added some comments

**Pull request closed without merging**